WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift

```diff
@@ -22,7 +22,7 @@ public class CurrencyFormatter {
         let unit = CurrencySettings.shared.symbol(from: currencyCode)
         newStringValue = newStringValue.replacingOccurrences(of: unit, with: "")
 
-        let decimalValue = NSDecimalNumber(string: newStringValue)
```

Comment:
@pmusolino – just want a 👍 from you on this merged change, if that's all right!

Reply:
Hi @jkmassel yes 👍  I substituted it with `let decimalValue = NSDecimalNumber(string: newStringValue, locale: Locale.current)` in this hotfix PR: https://github.com/woocommerce/woocommerce-ios/pull/1758